mycore.org/content/de/documentation/basics/resources/res_index.md

```diff
@@ -0,0 +1,250 @@
+- Sie dürfen keine leeren Segmente beinhalten.
+
+Die Pfade `/path/to/resource.txt` und `path/to/resource.txt` sind äquivalent. Der Pfad `/path//to/./some/../resources/` ist ungültig,
+weil er ein leeres Segment beinhaltet, die Segmente `.` und `..` beinhaltet und mit einem `/` endet.
+
+MyCoRe stellt die Klasse `MCRResourcePath` bereit, um Ressourcen-Pfade zu repräsentieren. Instanzen können z.B.
+mit den Methoden `MCRResourcePath#ofPath(String)` oder `MCRResourcePath#ofPathOrThrow(String)` erzeugt
+und anschließend z.B. mit `MCRresourceResolver#resolve(MCRResourcePath)` verwendet werden.
+
+> Die Repräsentation von Ressourcen-Pfaden ist bewusst simpel implementiert.
+> Sie hat kein Konzept davon, was es für Pfade in (verschiedenen Kontexten) bedeutet, „relativ zueinander zu sein“.
+> Code, der mit dem Resource-Resolver interagiert, ist dafür verantwortlich, absolute Pfade zu bilden.
+> Diese Einschränkung soll verhindern, dass Dateien ungewollt als Ressourcen zur Verfügung stehen.
+{.note}
+
+Außerhalb von Java-Code, z.B. in einem XSL-Stylesheet, können Ressourcen mit dem URI-Resolver ermittelt werden.
+Der URI-Resolver nutzt beim Auflösen der URI `resource:path/to/resource.txt` den Resource-Resolver mit dem Ressourcen-Pfad `/path/to/resource.txt`.
+
+## Web-Ressourcen
+
+*Web-Ressourcen* sind Ressourcen, deren Pfad mit `/META-INF/resources` beginnt.
+Dies entspricht der Konvention von Java-Servlet-Containern.
+
+Der Teil des Pfades einer Web-Ressource direkt inter dem Präfix `/META-INF/resources` ist der *Web-Pfad* dieser Web-Ressource.
+
+Die Ressource mit dem Pfad `/META-INF/resources/path/to/web-resource.txt` ist identisch mit der Web-Resource mit dem Web-Pfad `/path/to/web-resource.txt`.
+
+In einer klassischen Java-Web-Anwendung werden Web-Ressourcen mit Methoden wie `ServletContext#getResource(String)` genutzt.
+Diese Methoden nehmen einen Web-Pfad als `String` und liefern, sofern eine entsprechende Web-Ressource vorhanden ist, eine zugehörige URL als `URL` zurück.
+Über diese URL kann mit der eigentlichen Web-Ressource interagiert werden.
+
+Für Web-Pfade gelten in MyCoRe die folgenden Regeln:
+
+- Sie werden immer als absolut angenommen. Ein führender `/` ist optional.
+- Sie dürfen nicht mit `/META-INF` oder `/WEB-INF` beginnen.
+
+Der Ressourcen-Pfad `/META-INF/resources/META-INF/meta-resource.txt` ist ungültig, weil er den ungültigen Web-Pfad `/META-INF/meta-resource.txt` beinhaltet.
+
+Alle Methoden der Klassen `MCRResourceResolver`, `MCRResourceHelper` und `MCRResourcePath` haben jeweils eine alternative Variante,
+die direkt einen Web-Pfad entgegennimmt, z.B. `MCRResourceResolver#resolveWebResource(String)`, `MCRResourceHelper#getWebResourceUrl(String)` oder `MCRResourcePath#ofWebPath(String)`.
+Im Java-Code einer MyCoRe-Anwendung sollte es keinen Bedarf geben, den Präfix `/META-INF/resources` direkt zu verwenden.
+
```

Comment:
Wurde das im "Kern-Code" geprüft? Ich würde das in die Migrationsanleitung aufnehmen, damit ich (als Anwendungsentwickler) meinen Code diesbezüglich nochmal durchsehe.